content/en/monitors/guide/alert_aggregation.md

```diff
@@ -0,0 +1,89 @@
+
+### Simple Alert
+
+A Simple Alert monitor is a monitor that will aggregate your alerts into a single unique alert.
+{{< img src="monitors/guide/alert_aggregation/simple_alert_notification.png" alt="Notification configuration for a simple alert monitor" style="width:100%;" >}}
+
+Using the example, no matter which topic or partition breaches the threshold the monitor will alert. All notifications are aggregated into one single alert.
+
+For more information, see [Configure Monitors - Simple alert][1].
+
+### Multi Alert
+
+A Multi Alert monitor is a monitor that triggers for each unique combination of groups.  
+{{< img src="/monitors/guide/alert_aggregation/multi_alert_notification.png" alt="Notification configuration for a multi alert monitor" style="width:100%;" >}}
+In our case, the monitor will send a notification each time a combination of topic and partition breaches the threshold.  
+When using Multi Alert, using [Variables][2] become very useful to get more granularity in your notifications. They help you get a specific message depending on the group that triggers the alert.
+
+For more information, see [Configure Monitors - Multi alert][3].
+
+## Use cases
+
+{{% collapse-content title="Simple Alert" level="h4" expanded=false %}}
+#### Scenario
+You're monitoring a Kafka-based logging system with "error-logs" and "user-events". If any partition gets a message lag of more than 500, you want to know about it, but you don't need multiple alerts if multiple partitions are lagging.  
+
+#### Benefits of aggregation
+This is useful for teams that don't want excessive notifications but still need to act when issues arise.  
+However, if multiple partitions are lagging, you might not see every single affected partition in a single notification.
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Multi Alert" level="h4" expanded=false %}}
+Multi Alerts are great when a service is owned by multiple teams (e.g. each team is responsible for a dedicated component). Depending on which component is causing an issue, a different team should be notified.
+
+#### Scenario
+You're running an e-commerce order processing system, and messages are sent to the "order-events" topic.  
+  
+#### Benefits of aggregation
+If multiple partitions lag (e.g., partition 1 and partition 3), you need separate alerts because different partitions might correspond to different types of orders (e.g., domestic vs. international).  
+
+This level of detail helps engineers respond quickly and with precision.
+
+{{% /collapse-content %}}
+
+
+## How to do this via API
+
+If you are managing your monitors via the API, you want to use the variable "notify\_by" to make your monitor a Simple Alert or a Multi Alert.  
+
+To make the monitor a Simple Alert, you want to have "notify_by": [*].
+To make the monitor a Multi Alert, you want "notify_by": [<group>], for example: "notify_by": ["topic"].
```

Comment:
A table of this might be easier to read

```suggestion
| Type of Alert     | Configuration Example                  |
|-------------------|----------------------------------------|
| Simple Alert      | `"notify_by": [*]`                     |
| Multi Alert       | `"notify_by": [<group>]`, e.g., `"notify_by": ["topic"]` |
```